rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search/110_field_collapsing.yml

```diff
@@ -281,24 +281,6 @@ setup:
   - match: { hits.hits.1.fields.numeric_group: [1] }
   - match: { hits.hits.1.sort: [1] }
 
----
```

Comment:
Could we have a positive test case in yaml? This gives us way more coverage than IT tests.

Reply:
Added in https://github.com/elastic/elasticsearch/pull/107779/commits/c1fb5e33186a7142bd588e1345004538fc2ab98c